pkg/network/netconflist.go

```diff
@@ -2,6 +2,8 @@ package network
 
 import (
 	"net"
+	"os"
+	"path/filepath"
```

Comment:
Import ordering

Reply:
isnt it in the right spot ?

Reply:
Needs a newline between them

Reply:
it is not done that way in the other .go files we have.  path/filepath is in the with "main" group of imports.

Reply:
Nevermind, I'm just an idiot